src/SerilogTracing.Instrumentation.AspNetCore/SerilogTracing.Instrumentation.AspNetCore.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net8.0</TargetFramework>
+	    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
```

Comment:
Thanks! Still a minor issue here, it looks like the first indent is a tab character but the second is whitespace (this file's using four spaces instead of tabs).